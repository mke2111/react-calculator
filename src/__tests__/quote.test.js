import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../quote';

it('test the render of the Quote page', () => {
  const quote = renderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
