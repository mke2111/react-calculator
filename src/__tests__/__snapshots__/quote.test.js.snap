// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the render of the Quote page 1`] = `
<div
  className="midd"
>
  <h1>
    Quote of the day
  </h1>
  <p>
    This simple calculator is made with react.The project ensures proper use of routing in react
  </p>
</div>
`;
